resource-managers/yarn/src/test/scala/org/apache/spark/deploy/yarn/ClientSuite.scala

```diff
@@ -670,7 +670,7 @@ class ClientSuite extends SparkFunSuite with Matchers {
     ("files URI match test1", "file:///file1", "file:///file2"),
     ("files URI match test2", "file:///c:file1", "file://c:file2"),
     ("files URI match test3", "file://host/file1", "file://host/file2"),
-    ("wasb URI match test", "wasb://bucket1@user", "wasb://bucket1@user/"),
+    ("wasb URI match test", "wasb://user@bucket1", "wasb://user@bucket1/"),
```

Comment:
According to the Hadoop doc, https://hadoop.apache.org/docs/current/hadoop-azure/index.html#Accessing_wasb_URLs, the original uri is reasonable?

Reply:
It makes sense that wasb is a special case for host, but it does not violate the original uri. The main problem I'm referring to is that s3 writes the password in the wrong place, and I can't find a way to write it.

Reply:
wasb I'm going back to the standard format